methods_and_recursion/bottles_of_beer.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+
+def bottles(beer)
+  return 0 if beer == -1
+
+  beer != 1 ? word = 'bottles' : word = 'bottle'
+  (beer - 1) != 1 ? word_2 = 'bottles' : word_2 = 'bottle'
+
+  beer == 1 ? count_down = 'no more' : count_down = beer - 1
+
+  print_verse(beer, word, word_2, count_down)
+
+  bottles(beer - 1)
+end
+
+def print_verse(beer, word, word_2, count_down)
+  if beer != 0
+    comb = "#{beer} #{word}"
```

Comment:
`comb` needs a better variable name...

Also, I've seen other students create a method `pluralize('bottle', number_of_bottles)` which would return 1 bottle, 2 bottles or no more bottles based on the number in `number_of_bottles`.